app/governance/components/votingChart/voteGraph.tsx

```diff
@@ -0,0 +1,405 @@
+          <div className={styles.amountInfo}>
+            <div>
+              <Text size="x-sm">
+                {formatBalance(noVotes.toString(), 0, { short: true })}{" "}
+              </Text>
+            </div>
+            <div className={styles.icon}>
+              <Icon
+                icon={{
+                  url: "/tokens/canto.svg",
+                  size: {
+                    width: 12,
+                    height: 12,
+                  },
+                }}
+                themed
+              />
+            </div>
+          </div>
+          {noHeight == 0 && (
+            <div
+              style={{
+                height: 200,
+              }}
+            />
+          )}
+        </div>
+        <div className={styles.barContainer}>
+          {vetoHeight > 0 && (
+            <div
+              className={styles.graph}
+              style={{
+                height: vetoHeight,
+              }}
+            ></div>
+          )}
+          <div className={styles.amountInfo}>
+            <div>
+              <Text size="x-sm">
+                {formatBalance(vetoVotes.toString(), 0, { short: true })}{" "}
+              </Text>
+            </div>
+            <div className={styles.icon}>
+              <Icon
+                icon={{
+                  url: "/tokens/canto.svg",
+                  size: {
+                    width: 12,
+                    height: 12,
+                  },
+                }}
+                themed
+              />
+            </div>
+          </div>
+          {vetoHeight == 0 && (
+            <div
+              style={{
+                height: 200,
+              }}
+            />
+          )}
+        </div>
+        <div className={styles.barContainer}>
+          {abstainHeight > 0 && (
+            <div
+              className={styles.graph}
+              style={{
+                height: abstainHeight,
+              }}
+            ></div>
+          )}
+          <div className={styles.amountInfo}>
+            <div>
+              <Text size="x-sm">
+                {formatBalance(abstainVotes.toString(), 0, { short: true })}{" "}
+              </Text>
+            </div>
+            <div className={styles.icon}>
+              <Icon
+                icon={{
+                  url: "/tokens/canto.svg",
+                  size: {
+                    width: 12,
+                    height: 12,
+                  },
+                }}
+                themed
+              />
+            </div>
+          </div>
+          {abstainHeight == 0 && (
+            <div
+              style={{
+                height: 200,
+              }}
+            />
+          )}
+        </div>
+      </div>
+      <div className={styles.inforow}>
+        <div className={styles.voteOption}>
+          <div className={styles.circleContainer}>
+            <div
+              className={styles.circle}
+              style={{ backgroundColor: "green" }}
```

Comment:
I don't think these colors are supposed to be used.

Reply:
changed the color as per the design